tests/baselines/reference/tscWatch/programUpdates/deleted-files-affect-project-structure-2.js

```diff
@@ -110,10 +110,8 @@ Output::
 12:00:28 AM - File change detected. Starting incremental compilation...
 
 
-a/b/f1.ts(1,15): error TS2307: Cannot find module './f2'.
```

Comment:
Don't think this is correct, will take a look and let you know

Reply:
It resolves to f2.js and hence no error (without `--noImplicitAny`) can you please add `"--noImplicityAny"` at https://github.com/microsoft/TypeScript/blob/master/src/testRunner/unittests/tscWatch/programUpdates.ts#L404 and https://github.com/microsoft/TypeScript/blob/master/src/testRunner/unittests/tscWatch/programUpdates.ts#L432 and update the baseline. Thanks

Reply:
Updated!